### ISSUE TITLE
Edge 16, 17, toFormat ZZZZ and ZZZZZ returns a string with \u200e characters and whitespace

### ISSUE THREAD

**Issue by deldreth** [moment/luxon#374]
I produced this on the Luxon documentation site (https://moment.github.io/luxon/) through Edge 16 and 17 (via Browserstack, though this was found in the wild through user testing) and checked that the locale was en-US.

From within the console:
```
luxon.DateTime.local().toFormat('ZZZZ');
luxon.DateTime.local().toFormat('ZZZZZ');
encodeURIComponent(luxon.DateTime.local().toFormat('ZZZZ'));
```

Couldn't copy so I grabbed a screenshot.
![luxon](https://user-images.githubusercontent.com/233009/49252528-3f092800-f3f2-11e8-99cf-2a602d829acd.png)

**Comment by icambron** [comment]
Yeah, I think I see the problem. If I'm right, it's an easy fix. It should also *not* reproduce in Edge 18

**Comment by icambron** [comment]
Fixed in 7fdd27c, released in 1.8.2.

Thanks for the report!
